wear/src/main/java/io/homeassistant/companion/android/home/MainViewModel.kt

```diff
@@ -45,13 +61,29 @@ class MainViewModel @Inject constructor() : ViewModel() {
             isToastEnabled.value = homePresenter.getWearToastConfirmation()
             homePresenter.getEntities().forEach {
                 entities[it.entityId] = it
+                updateEntityDomains()
             }
```

Comment:
I would do this after the loop rather than after each iteration since this is just looping through a known list.